Add typing to URL (#1084)
aio-libs/yarl: CHANGES/1084.contrib.rst

Reviewer: Not sure if it's just a `contrib`. Having types is like a public API for dev experience, for the end-users of yarl, I think. Eh?

Perhaps, it needs to be renamed or symlinked to another category too.

Author: I made it a feature as well

Reviewer: Thanks! Marking the thread as resolved.

> Add typing to the ``URL`` object -- by :user:`bdraco`.

Reviewer: ```suggestion
Covered the :class:`~yarl.URL` object with types -- by :user:`bdraco`.
```

Author: Thanks
